src/pages/studyView/StudyViewUtils.spec.tsx

```diff
@@ -95,6 +97,7 @@ import { remoteData, toPromise } from 'cbioportal-frontend-commons';
 import { autorun, observable, runInAction } from 'mobx';
 
 import { AlterationTypeConstants, DataTypeConstants } from 'shared/constants';
+import { ClinicalDataCollection } from 'cbioportal-ts-api-client/src/generated/CBioPortalAPIInternal';
```

Comment:
this should just be 
```import { ClinicalDataCollection } from 'cbioportal-ts-api-client'```
IDE seems to get it wrong.  but we have a test that catches it

Reply:
For me this does not work for types of the Internal client. We should discuss how this should work.

Reply:
@pvannierop i think i see reason.  if you look at ```packages/cbioportal-ts-api-client/src/index.tsx``` there is a comment which explains that we need to manually export types from the internal client (to avoid clashes with the public client.  So that's why it's not working for you with my suggestion above.

Reply:
I have added the types manually to index.d.ts of the api client and updated all occurences of this problem in the code.